appeals/web/src/server/appeals/appeal-details/safety-risks/safety-risks.validator.js

```diff
@@ -0,0 +1,9 @@
+import { createValidator } from '@pins/express';
+import { body } from 'express-validator';
+
+export const validateChangeSafetyRisks = createValidator(
+	body('safetyRisksDetails')
+		.if(body('safetyRisksRadio').equals('yes'))
```

Comment:
is this conditional validation? This is really clean, lovely to have as an example so we don't always have to resort to custom validators!

Reply:
It is :)